daml-foundations/daml-ghc/ghc-compiler/src/DA/Daml/GHC/Compiler/Options.hs

```diff
@@ -67,8 +68,16 @@ data Options = Options
     -- ^ custom options, parsed by GHC option parser, overriding DynFlags
   , optScenarioService :: EnableScenarioService
     -- ^ Controls whether the scenario service is started.
+  , optScenarioValidation :: ScenarioValidation
+    -- ^ Controls whether the scenario service server runs all checks
+    -- or only a subset of them. This is mostly used to run additional
+    -- checks on CI while keeping the IDE fast.
   } deriving Show
 
+data ScenarioValidation
+    = ScenarioValidationLight
```

Comment:
Can we comment as to what is Light vs Full? My first question is why there is no None, which is of course what I want :)

Reply:
I honestly don’t know the answer to that, @remyhaemmerle-da can probably answer this better.

Reply:
As explained in the issue #128, the "Light" validation run the [Collision](https://github.com/digital-asset/daml/blob/master/daml-lf/spec/daml-lf-1.rst#name-collision-restriction) check.

We perform this check because :
- it is fast. It just check the definition ids
- it is not done in Haskell side.